src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -75,17 +90,69 @@ public void hasPerson_personWithSameIdentityFieldsInAddressBook_returnsTrue() {
         assertTrue(addressBook.hasPerson(editedAlice));
     }
 
+    @Test
+    public void removePerson_personToRemove_personRemoved() {
```

Comment:
perhaps renaming to removePerson_personToRemove_success() is better?

Reply:
ok, changed the name.